src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService service = new UserService();
+        User user = service.findByEmail(email);
+        if (user != null) {
+            return user.getPassword().equals(password);
+        }
         return false;
```

Comment:
```suggestion
        return user != null && user.getPassword().equals(password);
```

Reply:
what's wrong with my implementaion? just for understanding..

Reply:
nothing's technically wrong, it's a code style suggestion, when we deal with boolean expressions and need to return a boolean, there's no need to use `true` `false` literals - we can achieve the same with less code
// Instead of this
if (condition) {
    return true;
} else {
    return false;
}
// You can do this
return condition;

Reply:
thx